app/views/_post.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="post border p-4 rounded-md border-black">
+  <h3><%= post.title %></h3>
+  <p><%= post.text %></p>
+  <div class='my-4 flex justify-end items-center gap-4 text-sm'>
+    <p>Comments: <%= post.comments.count %></p>
+    <div class='my-4 flex justify-end items-center gap-4 text-sm'>
+      <p>
+        <%= button_to 'Like', like_user_post_path(@user, post), remote: true, method: :post, class: 'bg-blue-500 px-4 py-2 cursor-pointer rounded-md text-white', data: { post_id: post.id } %><span id='likes-count'>
```

Comment:
- [ ] Fantastic progress on your implementation! 🚀 Your hard work is evident, but let's ensure we cover a crucial requirement: the `like` functionality. 🤝 After reviewing the project requirements, it seems we might have missed incorporating this feature.

**Here's a screenshot of the requirement for your reference:** 
<img width="809" alt="Screenshot 2023-12-15 at 21 28 13" src="https://github.com/martinkarugaba/blog_app/assets/107192076/de5cf54b-8224-4357-be25-db922d7de863">


 - To enhance the user experience, it's important to implement the `like` functionality as outlined. This feature not only aligns with the project specifications but also adds a valuable element for user engagement.

Happy coding! 😊👨‍💻🚀

Reply:
Hi @Strangeal my like functionality is working, did you try clicking the like button